go/iceberg/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
maybe also worth add a label for all go related changes such as in https://github.com/apache/iceberg/blob/master/.github/labeler.yml

Reply:
Sounds good for me, does anyone have a preference of a label of "Go" vs "Golang" vs something else?

Reply:
I'd probably go with `Golang`

Reply:
Added